### DISCUSSION TITLE
贴吧的官号的UID

### DISCUSSION THREAD

**Discussion by adk23333** [adk23333/tieba-admin#21] [Ideas]
用于加入白名单，避免误处理

给你分享了贴吧号#52751290#整段复制后打开贴吧即可找到Ta

**Comment by n0099** [comment]
portrait，请

**Reply by adk23333** [reply]
> portrait，请

官方号的portrait也没什么特别的吧，我只是觉得UID方便我在手机上copy记录而已

**Reply by n0099** [reply]
鸡血神有`贴吧uid转其他id`的接口
但我没有所以我根本看不到这是哪个`@贴吧madam`
